realtime/utils.js

```diff
@@ -39,6 +39,17 @@ function isThisSubject(obj) {
 }
 
 /**
+ * A function to see if an object is a room object or not. It returns true
+ * if an object passed has 'type' and 'settings' as its property.
+ * @param  {Object}  obj - An object instance
+ * @returns {Boolean} - returns true if the object has the property
+ * "type" && "settings"
+ */
+function isThisRoom(obj) {
+  return obj.hasOwnProperty('type') && obj.hasOwnProperty('settings');
```

Comment:
again, feels like weak type check
again, function name
here, i guess you're not necessarily guaranteeing the object to be an instance of a sequelize model? what could you do to tighten the type check?

Reply:
same as above for the function name

Reply:
> same as above for the function name

regardless, please update